consent-management/consent-aspsp-api/src/main/java/de/adorsys/psd2/consent/aspsp/api/tpp/CmsAspspTppService.java

```diff
@@ -60,4 +61,14 @@
      * @return <code>true</code> if TPP was found and unlock was done. <code>false</code> otherwise.
      */
     boolean unblockTpp(@NotNull String tppAuthorisationNumber, @NotNull String nationalAuthorityId, @NotNull String instanceId);
+
+    /**
+     * Loads TPP info record by TPP ID
+     *
+     * @param tppId         ID of TPP to load
+     * @param instanceId    optional ID of particular service instance
+     * @return TPP info object if found in DB
+     */
+    @NotNull
+    Optional<TppInfo> getTppInfo(@NotNull String tppId, @NotNull String instanceId);
```

Comment:
here we need to add 'nationalAuthorityId' parameter because the uniqueness of the TPP depends also on  national authority
It will look like:
`
getTppInfo(@NotNull String tppAuthorisationNumber, @NotNull String nationalAuthorityId, @NotNull String instanceId) { 
`